README.md

```diff
@@ -101,6 +101,59 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
+# Development
+
+> This short guide is meant to give a guide to how github apps are made, and basic steps to reproduce the OpenCollective Bot should you wush to fix any bug.
+
+... Note Probot was used to build the Github app hence certain details may be specific to **Probot**
+
+## Installation
+
+Feel free to clone or download this project using the **git clone** and install using **Yarn install** or **Npm install** to install required dependencies
+
+Github apps make a large use of **WebHooks**. Hence, you can use [Ngrok](https://ngrok.com) , [Smee](https://smee.io) or other comfortable options .
+
+### Running Ngrok
+
+> After making all necessary local Ngrok installations found here [Ngrok](https://ngrok.com)
+
+- Run `Ngrok http <{localhost port}>` and copy the web address it gives into the your `.env` file
+
+### Registering App
+
+> Follow steps found here [Github](https://developer.github.com/apps/building-github-apps/creating-a-github-app/)
+
+#### Handling Credentials
+
+After registering the app, take note of the following listed details shown in the about page 
+
+- App ID
+- Client ID
+- Client Secret
+
+
+> Using a `.env` file is highly recommended and also including it in your `.gitignore` file.
```

Comment:
It's already in `.gitignore` isn't it?

Reply:
I forgot to check that out.  

I would remove this